src/main/java/ch/njol/skript/entity/PlayerData.java

```diff
@@ -1,110 +0,0 @@
-/**
- *   This file is part of Skript.
- *
- *  Skript is free software: you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
- *
- *  Skript is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
- *
- * Copyright Peter Güttinger, SkriptLang team and contributors
- */
-package ch.njol.skript.entity;
-
-import org.bukkit.Location;
-import org.bukkit.entity.Player;
-import org.bukkit.util.Consumer;
-import org.eclipse.jdt.annotation.Nullable;
-
-import ch.njol.skript.lang.Literal;
-import ch.njol.skript.lang.SkriptParser.ParseResult;
-
-/**
- * @author Peter Güttinger
- */
-public class PlayerData extends EntityData<Player> {
-	static {
-		EntityData.register(PlayerData.class, "player", Player.class, 1, "non-op", "player", "op");
```

Comment:
This class registers player EntityData therefore should not be deleted, only the `op` & `non-op` support should be deleted

Reply:
I register it as a simple entity data but the change is somewhere in the mess I made